plugins/plugins/opa/config.go

```diff
@@ -76,7 +76,10 @@ func (conf *config) Init(cb api.ConfigCallbackHandler) error {
 	ctx := context.Background()
 
 	query, _ := rego.New(
-		rego.Query(fmt.Sprintf("allow = data.%s.allow", policy)),
+		rego.Query(fmt.Sprintf(`
+            allow = data.%[1]s.allow;
+            custom_response = object.get(data.%[1]s, "custom_response", null)
+        `, policy)),
```

Comment:
Is it possible to add a schema so that we can check the type of custom_response when loading the policies?
OPA supports adding schema: https://www.openpolicyagent.org/docs/policy-language#schema-annotations, not sure if the library supports it.